src/components/SingleFeature/index.css

```diff
@@ -1,11 +1,25 @@
-.imgclass {
-  display: flex;
-  flex-flow: row wrap;
-  align-items: center;
-  justify-content: space-around;
+.blogContainer {
+  text-align: justify;
+  margin: 100px auto;
+  border: #6c567b;
```

Comment:
Please, use variables for colors and rem instead of px

Reply:
edited